models/mapper.go

```diff
@@ -49,6 +53,14 @@ func RectifyKeywords(keywords []string, keywordsLimit int) []string {
 	return rectifiedKeywords[:keywordsLimit]
 }
 
+func ValidateTopics(topics []string) []string {
+	if topics == nil {
+		log.Warn(context.Background(), "warning : topics not present, i.e. null")
```

Comment:
We shouldn't be creating another `context`. The `context` which is created when the service initially runs should be used. Therefore, this function should have a parameter called `ctx context.Context`